### PR TITLE
[CI-Examples] Update examples to use TOML-array syntax for files

### PR DESCRIPTION
**Pull request by dimakuv**
## Description of the changes 

All example manifests now use the TOML-array syntax for `sgx.trusted_files`, `sgx.allowed_files`, `sgx.protected_files`.

## How to test this PR? 

All examples should work.

---
This change is [Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)

### CODE REVIEW

**Review by mkow — APPROVED** [review]
> Reviewed 14 of 14 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)

___
*[-- commits, line 5 at r1](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj-Le_7xPUef5wvw3M):*
> ```
> , `sgx.protected_files`
> ```

I'd prefer the "," be replaced with "and" here, but I'm not blocking.

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: 13 of 14 files reviewed, all discussions resolved, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel) (waiting on @mkow)

___
*[-- commits, line 5 at r1](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj-Le_7xPUef5wvw3M):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> ```
> , `sgx.protected_files`
> ```

I'd prefer the "," be replaced with "and" here, but I'm not blocking.
</blockquote></details>

Done

**Review by mkow — DISMISSED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)

**Review by pwmarcz — CHANGES_REQUESTED** [review]
> Reviewed 13 of 14 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (1 more required) (waiting on @dimakuv)

___
*[CI-Examples/redis/redis-server.manifest.template, line 107 at r2](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj3dZNEgg2quK-EZgQ:-Mjj3dZNEgg2quK-EZgR:bayy5k8) ([raw file](https://github.com/gramineproject/gramine/blob/f9ff11091f8400912b7a8b3bbbd8ba03fdd6c7ff/CI-Examples/redis/redis-server.manifest.template#L107)):*
That's not strictly true (`sgx.trusted_files.sha256` is not a real table, like `sgx.trusted_checksum` before). I suggest rewording:

>As part of the build process, Graphene-SGX script (`graphene-sgx-sign`) finds each specified file, measures its hash, and **adds it to the manifest entry for that file (converting each entry to a table with "uri" and "sha256" keys).**

Then, later:

>At runtime, during loading of each "trusted file", Graphene-SGX measures its hash and compares with **the "sha256" value in the manifest entry.**
___
*[CI-Examples/redis/redis-server.manifest.template, line 126 at r2](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj6bcADCwok2counxO:-Mjj6bcADCwok2counxP:b-t5t1m4) ([raw file](https://github.com/gramineproject/gramine/blob/f9ff11091f8400912b7a8b3bbbd8ba03fdd6c7ff/CI-Examples/redis/redis-server.manifest.template#L126)):*
> ```
> # as hash mismatch by Graphene-SGX).
> #
> # Redis by default does not use configuration files, so this section is empty.
> ```

There's no separate section anymore, so I would remove this line, and move previous the paragraph somewhere above `sgx.trusted_files`.

**Review by dimakuv — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: 13 of 14 files reviewed, 2 unresolved discussions, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "fixup! " found in commit messages' one-liners (waiting on @mkow and @pwmarcz)

___
*[CI-Examples/redis/redis-server.manifest.template, line 107 at r2](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj3dZNEgg2quK-EZgQ:-MjmTIMQ5WosaYG4MOCJ:b-896fix) ([raw file](https://github.com/gramineproject/gramine/blob/f9ff11091f8400912b7a8b3bbbd8ba03fdd6c7ff/CI-Examples/redis/redis-server.manifest.template#L107)):*
<details><summary><i>Previously, pwmarcz (Paweł Marczewski) wrote…</i></summary><blockquote>

That's not strictly true (`sgx.trusted_files.sha256` is not a real table, like `sgx.trusted_checksum` before). I suggest rewording:

>As part of the build process, Graphene-SGX script (`graphene-sgx-sign`) finds each specified file, measures its hash, and **adds it to the manifest entry for that file (converting each entry to a table with "uri" and "sha256" keys).**

Then, later:

>At runtime, during loading of each "trusted file", Graphene-SGX measures its hash and compares with **the "sha256" value in the manifest entry.**
</blockquote></details>

Done.
___
*[CI-Examples/redis/redis-server.manifest.template, line 126 at r2](https://reviewable.io/reviews/gramineproject/gramine/66#-Mjj6bcADCwok2counxO:-MjmUThsDDJ72XH0Zmqa:b2e6s9r) ([raw file](https://github.com/gramineproject/gramine/blob/f9ff11091f8400912b7a8b3bbbd8ba03fdd6c7ff/CI-Examples/redis/redis-server.manifest.template#L126)):*
<details><summary><i>Previously, pwmarcz (Paweł Marczewski) wrote…</i></summary><blockquote>

There's no separate section anymore, so I would remove this line, and move previous the paragraph somewhere above `sgx.trusted_files`.
</blockquote></details>

I just removed this whole section. It is not very useful. Instead, I added a couple words in the first para.

**Review by pwmarcz — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required), "fixup! " found in commit messages' one-liners

**Review by mkow — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, all discussions resolved, "fixup! " found in commit messages' one-liners

**Review by mkow — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)

**Review by pwmarcz — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/66)* status: :shipit: complete! all files reviewed, all discussions resolved
